src/main/java/service/MyBatisEmployeeServiceImpl.java

```diff
@@ -0,0 +1,80 @@
+package service;
+
+import dao.IDao;
+import model.Employee;
+import model.Project;
+import mybatis.EmployeeMapper;
+import mybatis.ProjectMapper;
+import org.apache.ibatis.io.Resources;
+import org.apache.ibatis.session.SqlSession;
+import org.apache.ibatis.session.SqlSessionFactory;
+import org.apache.ibatis.session.SqlSessionFactoryBuilder;
+
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyBatisEmployeeServiceImpl implements IDao <Employee> {
+
+    private SqlSessionFactory sqlSessionFactory;
+
+    public MyBatisEmployeeServiceImpl() {
+        String resource = "mybatis-config.xml";
+        try {
+            InputStream inputStream = Resources.getResourceAsStream(resource);
+            sqlSessionFactory = new SqlSessionFactoryBuilder().build(inputStream);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
```

Comment:
Since `sqlSessionFactory` does not rely on the entity class (in this case 'Employee'), this could be done in a superclass and accessed through a getter.